scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SchedulerPage.java

```diff
@@ -321,75 +321,60 @@ public void onClick(ClickEvent clickEvent) {
-            public void onClick(MenuItemClickEvent event) {
-                FlatSubmitWindow w = new FlatSubmitWindow(SchedulerPage.this.controller);
-                w.show();
-            }
+        flatSubmitMenuItem.addClickHandler(event -> {
+            FlatSubmitWindow w = new FlatSubmitWindow(SchedulerPage.this.controller);
+            w.show();
         });
 
         MenuItem settingsMenuItem = new MenuItem("Settings", Images.instance.settings_16().getSafeUri().asString());
-        settingsMenuItem.addClickHandler(new com.smartgwt.client.widgets.menu.events.ClickHandler() {
-            public void onClick(MenuItemClickEvent event) {
-                SchedulerPage.this.settingsWindow.show();
-            }
-        });
+        settingsMenuItem.addClickHandler(event -> SchedulerPage.this.settingsWindow.show());
 
         MenuItem credMenuItem = new MenuItem("Create credentials", Images.instance.key_16().getSafeUri().asString());
-        credMenuItem.addClickHandler(new com.smartgwt.client.widgets.menu.events.ClickHandler() {
-            public void onClick(MenuItemClickEvent event) {
-                CredentialsWindow credentialsWindow = new CredentialsWindow();
-                credentialsWindow.show();
-            }
+        credMenuItem.addClickHandler(event -> {
+            CredentialsWindow credentialsWindow = new CredentialsWindow();
+            credentialsWindow.show();
         });
 
         MenuItem thirdPartyCredentialsMenuItem = new MenuItem("Manage third-party credentials",
                                                               Images.instance.key_16().getSafeUri().asString());
-        thirdPartyCredentialsMenuItem.addClickHandler(new com.smartgwt.client.widgets.menu.events.ClickHandler() {
-            public void onClick(MenuItemClickEvent event) {
-                ThirdPartyCredentialsWindow credentialsWindow = new ThirdPartyCredentialsWindow(SchedulerPage.this.controller);
-                credentialsWindow.show();
-            }
+        thirdPartyCredentialsMenuItem.addClickHandler(event -> {
+            ThirdPartyCredentialsWindow credentialsWindow = new ThirdPartyCredentialsWindow(SchedulerPage.this.controller);
+            credentialsWindow.show();
         });
 
         MenuItem serversMenuItem = new MenuItem("Data servers", Images.instance.server_16().getSafeUri().asString());
-        serversMenuItem.addClickHandler(new com.smartgwt.client.widgets.menu.events.ClickHandler() {
-            public void onClick(MenuItemClickEvent event) {
-                String url = GWT.getModuleBaseURL() + "servers?codebase=" + GWT.getHostPageBaseURL();
-                Window.open(url, "_blank", "");
-            }
+        serversMenuItem.addClickHandler(event -> {
+            String url = GWT.getModuleBaseURL() + "servers?codebase=" + GWT.getHostPageBaseURL();
+            Window.open(url, "_blank", "");
         });
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Make this anonymous inner class a lambda [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)